administrator/language/de-DE/com_contact.ini

```
@@ -27,7 +27,7 @@ COM_CONTACT_FIELD_ARTICLES_DISPLAY_NUM_LABEL="# gelistete Beiträge"
 COM_CONTACT_FIELD_ARTICLES_SHOW_LABEL="Beiträge"
 COM_CONTACT_FIELD_CAPTCHA_LABEL="Kontaktformular-<a href='https://de.wikipedia.org/wiki/CAPTCHA' target='_blank' rel='noopener noreferrer'>Captcha</a>"
 COM_CONTACT_FIELD_CATEGORY_LABEL="Kategorie auswählen"
-COM_CONTACT_FIELD_CONFIG_BANNED_EMAIL_DESC="Liste von E-Mail-Adressen, denen die Nutzung des Kontaktformulars nicht erlaubt ist. Mehrere E-Mail-Adressen müssen mit einem Semikolon getrennt werden."
+COM_CONTACT_FIELD_CONFIG_BANNED_EMAIL_DESC="Mehrere E-Mail-Adressen müssen mit einem Semikolon getrennt werden. E-Mail-Adressen von einer vollständigen Domäne können durch die Verwendung von @example.com oder einer Teildomäne durch ple.com gesperrt werden."
```

```suggestion
COM_CONTACT_FIELD_CONFIG_BANNED_EMAIL_DESC="Mehrere E-Mail-Adressen müssen mit einem Semikolon getrennt werden. Mit @example.com können E-Mails von einer gesamten Domäne gebannt werden oder von einer Teildomäne mit ple.com.""
```
Ist Teildomäne hier richtig?
"ple.com" umfasst doch eine größere Menge als "@example.com" 
oder mache ich hier einen Denkfehler?